admin/psql

```diff
@@ -33,8 +33,7 @@ if ($help) {
 my $db = Databases->get($key) or die "No such database '$key'\n";
 
 if ($system) {
-    my $sys_db = Databases->get('SYSTEM');
-    $db = $db->meta->clone_object($sys_db, database => $db->database);
+    $db = Databases->get('SYSTEM_' . $key);
 }
```

Comment:
For how to use this script, I appended the commit https://github.com/metabrainz/musicbrainz-server/pull/3079/commits/f41729a8c47b5c177b17a84f9f7bb5edff1ce836.